.circleci/config.yml

```diff
@@ -0,0 +1,136 @@
+version: 2.1
+
+
+commands:
+  npm-install:
+    steps:
+      - run: sudo npm install npm@6 --global
```

Comment:
I think Node 8 onwards ships with NPM 6, so this step is perhaps not nessecery? I don't see a real reason to update packages from the docker image, this can lead to less reproducable build because an update to NPM 6 could cause different behaviour upon CI restarts whereas relying on the NPM from the docker image will always be the same.